### ISSUE TITLE
Implement dry run version of the script

### ISSUE THREAD

**Issue by ka8725** [widefix/actual_db_schema#19]
Create a mechanism that allows to roll back the phantom migrations.

Ideas on how to implement it (can be implemented both):
1. Add an option to the command line: `rake db:rollback_branches --dry-run`
2. Via env var. Apply dry run if `ACTUAL_DB_SCHEMA_ROLLBACK_BRANCHES_DRY_RUN=true`

The dry run should not roll back any migrations but rather output the phantom migrations list.

Prefer the same output format in the output as in the following Rails' task:

```ruby
rails db:migrate:status

 Status   Migration ID    Migration Name
--------------------------------------------------
   up     20220923211114  Initial
   up     20221003092332  Add typical clients and samples links to vendor applications
   up     20221003094301  Add vendor id index to messages
   up     20221004031502  Add vendor company to users
   up     20221115233057  Add status to vendor applications
   up     20221129184509  Add type to quotes
   up     20221207181413  Create admin user approvals
   up     20221223154500  Add templates for sale to vendor application
   up     20230105173250  Create categories
   up     20230113173555  Add data to projects
   up     20230120192110  Create services
   up     20230126045226  Add terms accepted at to users
   up     20230131174014  Add active to services
```

**Comment by vic778** [comment]
Hello @ka8725, could you please provide some descriptions about the issue? I want to work on it.

**Comment by ka8725** [comment]
@vic778 done! Thanks for helping with that!

**Comment by vic778** [comment]
@ka8725 sorry for the delay I have a bunch of PRs to finish before starting my vacation. I will check on a new issue and work on it. Thanks for the update. I appreciate that

**Comment by ka8725** [comment]
Turned out this task was not needed. This issues covers it - https://github.com/widefix/actual_db_schema/issues/34
